x-pack/plugins/search_playground/public/components/summarization_panel/instructions_field.tsx

```diff
@@ -10,15 +10,20 @@ import React from 'react';
 import { EuiFormRow, EuiIcon, EuiTextArea, EuiToolTip } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { isEmpty } from 'lodash';
+import { useUsageTracker } from '../../hooks/use_usage_tracker';
+import { AnalyticsEvents } from '../../analytics/constants';
 
 interface InstructionsFieldProps {
   value?: string;
   onChange: (value: string) => void;
 }
 
 export const InstructionsField: React.FC<InstructionsFieldProps> = ({ value, onChange }) => {
-  const handlePromptChange = (e: React.ChangeEvent<HTMLTextAreaElement>) =>
+  const usageTracker = useUsageTracker();
+  const handlePromptChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
     onChange(e.target.value);
+    usageTracker.click(AnalyticsEvents.instructionsFieldChanged);
```

Comment:
this is called everytime a keypress. Is there a way to move this on blur instead?